.test-infra/metrics/grafana/dashboards/perftests_metrics/ParDo_Load_Tests.json

```diff
@@ -619,5 +855,5 @@
   "variables": {
     "list": []
   },
-  "version": 2
+  "version": 8
 }
```

Comment:
nit: add a newline?

Reply:
This file is auto-generated but I could add a newline :)